src/dotless.Benchmark/Program.cs

```diff
@@ -23,19 +23,29 @@ public static void Main(string[] args)
                 (file, engine) => Enumerable
                                       .Range(0, rounds)
                                       .Select(i =>
-                                                  {
-                                                      var starttime = DateTime.Now;
-                                                      engine.TransformToCss(contents[file], file);
-                                                      var duration = (DateTime.Now - starttime);
-                                                      return duration.Milliseconds;
-                                                  })
+                                      {
+                                          engine.ResetImports();
```

Comment:
Also this leads to these cases where you have to remember to call ResetImports()